packages/manager/apps/pci-private-network/src/api/hooks/useDatagridColumn.tsx

```diff
@@ -0,0 +1,71 @@
+import { DatagridColumn, DataGridTextCell } from '@ovhcloud/manager-components';
```

Comment:
this hook could be moved in the same directory as the listing page since it's more related to ui than api